.circleci/config.yml

```
@@ -104,28 +103,54 @@ jobs:
             - /tmp/docker
 
   get_data:
-    machine:
-      # Ubuntu 14.04 with Docker 17.10.0-ce
-      image: circleci/classic:201711-01
-    working_directory: /home/circleci/data
+    docker:
+      - image: continuumio/miniconda3:4.9.2-alpine
+    working_directory: /tmp/data
+    environment:
+      - TEMPLATEFLOW_HOME: /tmp/templateflow
     steps:
       - restore_cache:
           keys:
-            - data-v0-{{ .Branch }}-{{ .Revision }}
-            - data-v0-{{ .Branch }}
-            - data-v0-
+            - env-v0-{{ .Branch }}-
+            - env-v0-master-
+            - env-v0-
+      - run:
+          name: Setup git-annex & DataLad
+          command: |
+            conda install -y -c anaconda -c conda-forge git-annex datalad
+            python -m pip install --no-cache-dir -U datalad-osf
+            git config --global user.name 'NiPreps Bot'
+            git config --global user.email 'nipreps@gmail.com'
       - run:
-           name: Get single echo rodent data
-           command: |
-             mkdir -p /tmp/data
-             rm -rf /tmp/data/rodent-se/
-             wget --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 -t 0 -q \
-               -O rodent-se.tar.gz "https://stanford.box.com/shared/static/ei6qtu81y1zx9fxii11svvm73n3884fl.gz"
-             tar xvzf rodent-se.tar.gz -C /tmp/data/
+          name: Setup TemplateFlow
+          command: |
+            python -m pip install --no-cache-dir -U templateflow
+            python -c "from templateflow import api as tfapi; \
+                       tfapi.get('OASIS30ANTs', resolution=1, desc=None, suffix='T1w'); \
+                       tfapi.get('OASIS30ANTs', resolution=1, desc='BrainCerebellumExtraction', suffix='mask'); "
```

we're not using the OASIS template here
```suggestion
                       tfapi.get('Fischer344', desc=None, suffix='T2w'); \
                       tfapi.get('Fischer344', desc='BrainCerebellumExtraction', suffix='mask'); "
```